tools/packaging/build/agent-enclave-bundle/Dockerfile

```diff
@@ -1,22 +1,13 @@
-FROM rust:1.63-bullseye as builder
+FROM ubuntu:20.04 as builder
```

Comment:
Given that we make an one-stage building, `as builder` can be removed.

Reply:
good comment. I think we could keep it because we may want to go back to a multi-stage build where in the second stage only the instance and runtime dependencies get installed.

Reply:
Sure.